content/docs/cml/self-hosted-runners.md

```diff
@@ -83,27 +83,27 @@ you do it your way :)
+| ------------------------ | --------------------------------------------------------------------- | ---------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `--version`              | Flag                                                                  | Unset                                                      | Show version number and exit.                                                                                                |
+| `--labels`               | List                                                                  | `cml`                                                      | Comma delimited list of runner labels.                                                                                       |
+| `--idle-timeout `        | Integer                                                               | `300`                                                      | Time in seconds for the runner to be waiting for jobs before shutting down. Setting it to `0` disables automatic shutdown.   |
+| `--name`                 | String                                                                | `cml-{identifier}` where `{identifier}` is a random string | Name displayed in the repository once registered.                                                                            |
+| `--single`               | Flag                                                                  | Unset                                                      | Exit after running a single job.                                                                                             |
+| `--reuse`                | Flag                                                                  | Unset                                                      | Don't launch a new runner if an existing one has the same name or overlapping labels.                                        |
+| `--driver`               | Choice: `github` or `gitlab`                                          | Inferred from environment variables                        | Driver for the platform where the repository is hosted.                                                                      |
+| `--repo`                 | String                                                                | Inferred from environment variables                        | Git repository address (e.g. https://github.com/iterative/cml)                                                               |
+| `--token`                | String                                                                | Inferred from environment variables                        | Personal access token with enough permissions to register a self-hosted runner on the repository.                            |
```

Comment:
> Inferred from environment variables

Where are these env vars explained? Just curious 🙂

Reply:
I fear being unable to satisfy your curiosity. 😄 

We have them [sprinkled](https://github.com/iterative/cml/blob/5888e49d55662aafd20f62d9acf5ecc2c5478a43/bin/cml-runner.js#L16-L29) around [the code](https://github.com/iterative/cml/blob/81e8a363eb300306c888a77bdeb4d71179f09203/src/cml.js#L49-L55) in order to automatically detect the repository URL, the continuous integration platform and the default token it exposes to every workflow run. They aren't intended for user consumption, but a bit of documentation wouldn't hurt either.

Reply:
Cool.

- [ ] Maybe something else to consider documenting in a single place in the future.